File: BubbleSort.java
import java.util.Arrays;  
import java.util.Random;

class BubbleSort
{
    void bubbleSort(int arr[])
    {
        int n = arr.length;
        for (int i = 0; i < n-1; i++)
            for (int j = 0; j < n-i-1; j++)
                if (arr[j] > arr[j+1])
                {
                    int temp = arr[j];
                    arr[j] = arr[j+1];
                    arr[j+1] = temp;
                }
    }
 
    public static void main(String args[])
    {
        BubbleSort ob = new BubbleSort();
        int array[] = new int[5];
        for (int i = 0; i < array.length; i++) {
            array[i] = new Random().nextInt(100);
         }
        System.out.println(String.format("Unsorted array: %s", Arrays.toString(array)));

        ob.bubbleSort(array);
        System.out.println(String.format("Sorted array: %s", Arrays.toString(array)));
    }
}

File: BubbleSort.kt
import kotlin.random.Random
import java.util.Arrays

fun bubbleSort(array: IntArray): IntArray{
    var swap = true
    while (swap) {
        swap = false
        for (i in 0 until array.size - 1) {
            if (array[i] > array[i + 1]) {
                val temp = array[i]
                array[i] = array[i + 1]
                array[i + 1] = temp

                swap = true
            }
        }
    }
    return array
}

fun main() {
    val array = IntArray(5) { (1..100).random() }
    println("Unsorted array: " + Arrays.toString(array))
    bubbleSort(array)
    println("Sorted array: " + Arrays.toString(array))
}